.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: scala
+scala:
+   - 2.11.8
+   - 2.12.0-M4
```

Comment:
Yes, that's why it's in `allow_failures`, IMHO it's good to know how things look in advance before the new version comes out, but I may be approaching it wrong.